caches.py

```diff
@@ -100,3 +130,52 @@ def update(self, prediction):
         self.lock.release()
         if not exists:
             raise errors.PredictionNotFound
+
+    def invalidate(self):
+        self.lock.acquire()
+        self.data = {}
+        self.lock.release()
+
+
+class InfinispanCache(Cache):
+
+    def __init__(self, host, name, port):
+        self._host = host
+        self._name = name
+        self._port = port
+        self.invalidate()  # invalidate cache, just in case we have stale persisted JDG entries
+
+    def _connect(self):
+        return httplib.HTTPConnection(self._host, self._port)
+
+    def store(self, prediction):
+        conn = self._connect()
+        conn.request(method="POST", url=self._format(prediction['id']), body=json.dumps(prediction),
```

Comment:
i think we should check the response code here just to ensure we don't have an error